polyjuice/filch.py

```diff
@@ -21,7 +21,10 @@ def mount_iso(self, iso_path, out):
 
         return mount_point
 
-    def scrub(self, image, modifications, id_pairs, log, unknown_ids):
+    def scrub(self, image, modifications, id_pairs, log, unknown_ids, metadata_path):
+        if metadata_path:
```

Comment:
I dont like you putting the metadata file write in the scrub function. Its a side effect that you would not expect to happen when calling scrub